examples/LGACSend.ino

```diff
@@ -0,0 +1,258 @@
+// Copyright 2017 xpokor22
```

Comment:
Please move this to it's own directory. The Arduino IDE expects it like that.
e.g. examples/LGACSend/LGACSend.ino

Also, please add it to the .travis.yml file so it is compiled/tested etc.
e.g.
Add the following:
```
  - arduino --verify --board $BD $PWD/examples/LGACSend/LGACSend.ino
```